src/Raven.Server/Documents/DocumentFlags.cs

```diff
@@ -9,24 +9,17 @@ public enum DocumentFlags
 
         Artificial = 0x1,
         Versioned = 0x2,
-        Reserved1 = 0x4,
-        Reserved2 = 0x8,
+        Revision = 0x4,
+        Reserved = 0x8,
 
         FromIndex = 0x10,
-        FromVersionStorage = 0x20,
+        Reserved1 = 0x20,
         FromReplication = 0x40,
-        Reserved3 = 0x80,
+        FromSmuggler = 0x80,
 
-        HasAttachments = 0x100
-    }
-
-    [Flags]
-    public enum NonPersistentDocumentFlags
```

Comment:
I was taking about saving a document without creating revisions in a daily use from the client, not the smuggler import.
But yes, I'm OK with this. And you're right about users will be confused because of the flags. Maybe we can filter them out using a Mask?